src/service/pricing.rs

```diff
@@ -0,0 +1,28 @@
+use chatgpt::prelude::ChatGPTEngine;
+
+const INPUT_TOKEN_PER_USD_GPT3_5: f32 = 0.0015;
+const OUTPUT_TOKEN_PER_USD_GPT3_5: f32 = 0.002;
+const INPUT_TOKEN_PER_USD_GPT4: f32 = 0.03;
+const OUTPUT_TOKEN_PER_USD_GPT4: f32 = 0.06;
+const EXCHANGE_RATE_USD_PER_JPY: f32 = 142.0;
```

Comment:
```suggestion
const EXCHANGE_RATE_USD_PER_JPY: f32 = 150.0;
```

クレジットカード会社に 3.6% ぐらいの手数料を取られるのが世の常なので、実際はこれぐらいとられます

Reply:
¥ 150 は盛ってるけどまぁいいでしょう

Reply:
なるほど......そうなると￥150の方が良いですね、修正します